contracts/SortitionPool.sol

```diff
@@ -39,25 +39,57 @@ contract SortitionPool is SortitionTree {
     );
   }
 
-  /// @notice Add an operator to the pool,
+  /// @notice Inserts an operator to the pool,
   /// reverting if the operator is already present.
-  function joinPool(address operator) public {
+  /// @dev Can be called only by the contract owner.
+  /// @param operator Address of the inserted operator.
+  function insertOperator(address operator) public {
```

Comment:
This function could return operator ID. Non-blocking though, we can add it later once it's really needed.